packages/components/pie-button/README.md

```diff
@@ -65,13 +65,13 @@ import { PieButton } from '@justeattakeaway/pie-button/dist/react';
 
 ## Props
 
-| Property    | Type      | Default         | Description                                                                                                    |
-|-------------|-----------|-----------------|----------------------------------------------------------------------------------------------------------------|
-| size        | `String`  | `medium`        | Size of the button, one of `buttonSizes` – `xsmall`, `small-expressive`, `small-productive`, `medium`, `large` |
-| type        | `String`  | `submit`        | Type of the button, one of `buttonTypes` – `submit`, `button`, `reset`, `menu`                                 |
-| variant     | `String`  | `primary`       | Variant of the button, one of `buttonVariants`– `primary`, `secondary`, `outline`, `ghost`                     |
-| disabled    | `Boolean` | `false`         | If `true`, disables the button.                                                                                |
-| isFullWidth | `Boolean` | `false`         | If `true`, sets the button width to 100% of it's container.                                                    |
+| Property    | Type      | Default         | Description                                                          |
+|-------------|-----------|-----------------|----------------------------------------------------------------------|
+| size        | `String`  | `medium`        | Size of the button, one of `sizes` |
```

Comment:
Would be good to keep the different sizes/types/variants in the README, so that consumers don't have to dig into the code to see the different options.